src/effects/builtin/echoeffect.cpp

```diff
@@ -47,14 +47,14 @@ EffectManifestPointer EchoEffect::getManifest() {
     delay->setShortName(QObject::tr("Time"));
     delay->setDescription(QObject::tr(
         "Delay time\n"
-        "1/8 - 2 beats if tempo is detected\n"
-        "1/8 - 2 seconds if no tempo is detected"));
+        "1/8 - 4 beats if tempo is detected\n"
+        "1/8 - 4 seconds if no tempo is detected"));
     delay->setControlHint(EffectManifestParameter::ControlHint::KNOB_LINEAR);
     delay->setSemanticHint(EffectManifestParameter::SemanticHint::UNKNOWN);
     delay->setUnitsHint(EffectManifestParameter::UnitsHint::BEATS);
     delay->setMinimum(0.0);
     delay->setDefault(0.5);
-    delay->setMaximum(2.0);
```

Comment:
This should actually become a preference setting. Extending the maximum range would not affect users that prefer to stick with the current, hard-coded default setting of 2 bars.

I mostly use the center position = 1 beat. Setting the knob to exactly 1 beat becomes much harder after this change. Much worse than that: After starting Mixxx the knob is still at the center position, i.e. now resulting in a 2 beat echo. Silently modifying existing settings is not acceptable.

Reply:
> This should actually become a preference setting.

This will be possible after #1705 is merged with custom per-effect snapshots of parameter values and parameter ranges. Then we can set the absolute minimum and maximum values very wide and ship Mixxx with narrowed ranges by default. I do not think we should hack a preference option for this one effect before that more general solution is available.

> I mostly use the center position = 1 beat. Setting the knob to exactly 1 beat becomes much harder after this change.

IIRC this is why I chose 2 beats as the range in #1256.